R/legend-draw.R

```diff
@@ -252,9 +252,31 @@ draw_key_text <- function(data, params, size) {
 #' @export
 #' @rdname draw_key
 draw_key_label <- function(data, params, size) {
-  grobTree(
-    draw_key_rect(data, list()),
-    draw_key_text(data, list())
+  if(is.null(data$label))         data$label <- "a"
+  if(is.null(data$label.r))       data$label.r <- unit(0.1, "snpc")
+  if(is.null(params$label.padding)) params$label.padding <- unit(0.25, "lines")
+  if(length(params$label.padding) == 1) params$label.padding[2:4] <- params$label.padding
+
+  labelGrob(
+    label = data$label,
```

Comment:
Might skip all the `if (is.null(foo$bar)) foo$bar <- "baz"` above if you use `bar = foo$bar %||% "baz"` in setting the arguments of `labelGrob()`.

Reply:
Yeah, makes sense. Done in an upcoming commit.